Sources/ChatUI/Essentials/GiphyConfiguration.swift

```diff
@@ -0,0 +1,28 @@
+import Foundation
+import GiphyUISDK
+import SwiftUI
+
+public class GiphyConfiguration {
```

Comment:
It's delivered as a parameter to `ChatConfiguration`'s initializer and never change because `ChatConfiguration/giphyConfig` is declared as a constant. (Also the all properties in `GiphyConfiguration` are also not allowed to be changed.)
IMO, using `struct` instead of `class` is much better!

Reply:
```suggestion
public struct GiphyConfiguration {
```